components/gc-subway/gc-subway.js

```diff
@@ -44,14 +44,17 @@ var $document = wb.doc,
 			$h1 = $( "h1", $elm );
 			h1 = $h1.get( 0 );
 
-			// Ensure the element have an ID
-			if ( !h1.id ) {
-				h1.id = wb.getId();
-			}
+			if( h1 ) {
```

Comment:
Improve this condition, the 2 sub-instruction are only "if" conditional. For example `if ( h1 && !skipLink )`. But wait, is the variable "skipLink" is required? Is the init are called twice?

Reply:
@duboisp Yes, init gets called multiple times and this was causing multiple links to be added to the Skip links.